voluncheer/voluncheer/settings.py

```diff
@@ -165,14 +165,15 @@
 CRISPY_TEMPLATE_PACK = "bootstrap5"
 
 # Backend Email (testing)
-if os.getenv("IS_PRODUCTION"):
-    EMAIL_BACKEND = "django_ses.SESBackend"
-    AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY")
-    AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
-    AWS_SES_REGION_NAME = "us-east-2"
-    AWS_SES_REGION_ENDPOINT = "email.us-east-2.amazonaws.com"
-else:
-    EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+# if os.getenv("IS_PRODUCTION"):
+EMAIL_BACKEND = "django_ses.SESBackend"
+AWS_ACCESS_KEY_ID = os.getenv("SES_AWS_ACCESS_KEY")
+AWS_SECRET_ACCESS_KEY = os.getenv("SES_AWS_SECRET_ACCESS_KEY")
```

Comment:
```suggestion
AWS_SES_SECRET_ACCESS_KEY = os.getenv("AWS_SES_SECRET_ACCESS_KEY")
```

https://github.com/django-ses/django-ses#getting-going